### ISSUE TITLE
PORTAL: Add menu assignment tab

### ISSUE THREAD

**Issue by Lowell-Syncrety** [DeskDirector/UserVoice#452]
When I go to add a new company to say, the Bigger Brains menu I naturally go to Menus, and I see that there are X companies/contacts configured to use this menu but can't click on them to manage which companies are listed.

So I click on the custom menu, and there's no longer even a total of how many companies/contacts are assigned to this menu. In this space I'd like to see a tab where I can go and assign companies/contacts to this menu, and do it for more then one company at a time similar to how we could do it before. The only way I currently can find to do this is by going to Companies (or Contacts), search for the company, click on the company, go to Menu, and change their menu.

**Comment by Nness** [comment]
Menu currently is cascading system. It is not same as learning, where it is filter based.

What we are planning is bulk editing, where inside companies page, you can select multiple entities and assign or unassign settings to them.

e.g. select all contacts from a company, unassign menu to those contacts. Then go to company list, select multiple companies and assign new menu to them.

The way you described won't work for cascading system. Here is scenario.

e.g. Custom Menu 1 been assigned to user xxx that's belong to company A.

Custom menu 2, assigned to company A. You think user xxx will see menu 2?

Actually no, user xxx will see custom menu 1. 

From menu interface, you cannot tell whether there are specific users that won't respect those company or global level configuration.

**Comment by Lowell-Syncrety** [comment]
Yeah, I figured that as I was typing up my request, but thought you'd figure something out for bulk editing regardless of whether the underlying structure was the same or vastly different. Sounds like you're on it, thanks!

**Comment by Olivia-Lancom** [comment]
[49139] Way to apply changes in Custom Menu in bulk in the Client Portal
